Apps/Playground/Scripts/experience.js

```diff
@@ -158,6 +159,18 @@ CreateBoxAsync().then(function () {
             });
         }, 5000);
     }
+
+    if (text) {
+        var Writer = BABYLON.MeshWriter(scene, { scale: 1.0, defaultFont: "Arial" });
+        text1 = new Writer(
```

Comment:
Where is `text1` used?

Reply:
It isn't; copy-paste leftover. I'll remove it.

Reply:
I’m confused then... how do you set the transform of the text? How do you remove the text from the scene?

Reply:
I should have clarified. You _can_ do all those things using the former `text1` variable or equivalent; you can get an ordinary Babylon mesh from it with which you can do whatever you want. (The "usage summary" on the NPM page you linked to above shows an example of this.) I just don't do any of that here, so the variable was never used.